partitura/score.py

```diff
@@ -2400,22 +2401,27 @@ class Measure(TimedObject):
 
     Parameters
     ----------
-    number : int or None, optional
-        The number of the measure. Defaults to None
+    number : int
+        The running count independent of measure regularity/ volta endings, continuously counting up all measures in a musicxml score file and always starting from one.
+    name : string, optional
+        The ID of the measure in a given musicxml score file. Can be a non-number in case of volta endings, irregular measures (i.e., pickup measures in the middle of the piece). Defaults to None
 
     Attributes
     ----------
-    number : intp
+    number : int
+        See parameters
+    name : str
         See parameters
 
     """
 
-    def __init__(self, number=None):
+    def __init__(self, number=None, name=None):
         super().__init__()
         self.number = number
+        self.name = name
 
     def __str__(self):
-        return f"{super().__str__()} number={self.number}"
+        return f"{super().__str__():16s} number={self.number} name={self.name}"
```

Comment:
Why is the addition of the spaces necessary (the `:16s`)? Is it because of a specific formatting required by `pretty_print`?